cmd/cluster/core/create.go

```diff
@@ -70,6 +70,7 @@ func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opts.ControlPlaneAvailabilityPolicy, "control-plane-availability-policy", opts.ControlPlaneAvailabilityPolicy, "Availability policy for hosted cluster components. Supported options: SingleReplica, HighlyAvailable")
 	flags.BoolVar(&opts.Render, "render", opts.Render, "Render output as YAML to stdout instead of applying")
 	flags.StringVar(&opts.RenderInto, "render-into", opts.RenderInto, "Render output as YAML into this file instead of applying. If unset, YAML will be output to stdout.")
+	flags.BoolVar(&opts.RenderSensitive, "render-sensitive", opts.RenderSensitive, "Render sensitive information in the output")
```

Comment:
It looks like RenderSensitive defaults to false now. Any chance of defaulting this to true so it won't break existing jobs which make use of the create - patch - apply logic ? 

Maybe we could negate the flag by making it `--no-render-sensitive`.

Reply:
Where does it default to false, from my understanding the is no default value and we only set it to false for azure e2e.

Reply:
It is a bool so its value will be false unless we set it to true